### PR TITLE
Fix empty string bug in createAnnouncmentString()

### PR DESCRIPTION
**Pull request by PorterNan**
# What

The 2 sets of array - unnamedParticipants + namedParicipants should exactly match the original participants array.
But there is one case missing - participantName === '' 
So if any empty string participant appears, this bug will cause unexpected exception.

Change the condition a bit to include '' empty string in unnamedParticipant
# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by edwardlee-msft** [comment]
Create mini fix! 
Note for other reviewers:
Empty strings are "falsy" which means they are considered false in a Boolean context, so you can just use not string

**Review by edwardlee-msft — APPROVED** [review]

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ export const createAnnouncmentString = (
    * Filter participants into two arrays to put all the unnamed participants at the back of the
    * names being announced.
    */
-  const unnamedParticipants = participants.filter((p) => p.displayName === undefined);
+  const unnamedParticipants = participants.filter((p) => !p.displayName);
```
Comment (dmceachernmsft) [inline packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.ts:103]: How does this also cover the case when the `displayName === ''`?

Comment (PorterNan) [reply packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.ts:103]: This might feel a bit strange, but in js  ```!'' = true```

Comment (PorterNan) [reply packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.ts:103]: @edwardlee-msft 's comment above explains this in a more detailed way 👍

**Pull request merged**